databallpy/filters.py

```diff
@@ -0,0 +1,62 @@
+import pandas as pd
+from scipy.signal import savgol_filter
+
+
+def filter_data(
+    df: pd.DataFrame,
+    input_columns: list,
+    kind: str,
```

Comment:
Maybe refactor this to `filter_type` or something. I think `kind` is not specific enough.